services/submission/src/test/java/app/coronawarn/server/services/submission/controller/SubmissionControllerTest.java

```diff
@@ -253,17 +253,33 @@ private TemporaryExposureKey createOutdatedKey() {
   }
 
   private void assertElementsCorrespondToEachOther
-      (Collection<TemporaryExposureKey> submittedKeys, Collection<DiagnosisKey> keyEntities) {
-    Set<DiagnosisKey> expKeys = submittedKeys.stream()
-        .map(aSubmittedKey -> DiagnosisKey.builder().fromProtoBuf(aSubmittedKey).build())
+      (Collection<TemporaryExposureKey> actTemporaryExposureKeys, Collection<DiagnosisKey> expDiagnosisKeys) {
```

Comment:
I think I mixed up actual/expected and saved/submitted in the variable naming and testing logic. Will fix this tomorrow.

Reply:
Fixed